lib/utils/helpers.coffee

```diff
@@ -52,3 +55,24 @@ exports.sudo = (command, message) ->
 		console.log('Type your computer password to continue')
 
 	return president.executeAsync(command)
+
+# Loads '.resin-sync.yml' configuration from 'source' directory.
+# Returns the configuration object on success
+#
+# TODO: Use 'config.load()' method from `resin sync` when resin sync gets
+# integrated into resin CLI
+exports.loadConfig = (source) ->
+	configPath = path.join(source, '.resin-sync.yml')
```

Comment:
We shouldn't hardcode `.resin-sync.yml` here if we call the function `loadConfig()`, since that implies `.resin-sync.yml` is the configuration for the CLI as a whole. I propose either:

- Accepting the configuration file name as an argument.
- Changing the name to `loadSyncConfig()`.
- Moving this function to a private function in `sync.coffee`.

I personally prefer the third one.